.github/PROCESS.md

```diff
@@ -85,6 +86,57 @@ We have two long-living branches:
 - `master`: completed features, bug fixes, refactors, chores
 - `stable`: the latest release
 
+The overall flow:
+
+1. A `release` branch is created from `master`
+1. Feature, refactor, and bug fix branches are created from `master`
+1. When a feature, refactor, or fix is complete it is merged into `master`
+1. When the release branch is done it is merged into `master` and `stable`
+1. If an issue in `stable` is detected a `hotfix` branch is created from `stable`
+1. Once the `hotfix` is complete it is merged to both `master` and `stable`
+1. All branches should follow the syntax of `{type}-{details}` where `{type}` is the type of branch (`hotfix`, `release`, or one of the [commit types](https://github.com/ionic-team/ionic/blob/master/.github/CONTRIBUTING.md#commit-message-format)) and `{details}` is a few hyphen separated words explaining the branch
+
+
+### Stable and Master Branches
+
+#### Stable Branch
+
+Branches created from `stable`:
+
+- A `hotfix` branch (e.g. `hotfix-missing-export`) should be the *only* branch that is created from stable and it gets merged back into both `master` and `stable` when complete
+
+#### Master Branch
+
+Branches created from `master`:
+
+The following branches should be created from `master` and merged back to `master` via a pull request:
+
+1. A feature branch (e.g. `feat-desktop-support`): an addition to the API that is not a bug fix or regression fix
+1. A bug fix branch (e.g. `fix-tab-color`): a bug fix that is not fixing a regression or issue with a published release
```

Comment:
Where do regressions fit into this? even if we found a regression like a week after a release, would that still be considered a hotfix?

Reply:
This is a good question. @jthoms1 what do you think? We have regressions from versions such as v3 - do those warrant a hotfix or is it just if it's breaking something?

Reply:
I think this is up to the situation. I don't think we should prescribe a hotfix anytime there is a regression, just know that it is available if we need to use it.

Reply:
I have to agree with Josh on this. I think this should be up to our discretion. I don't think any regressions with known workarounds should be considered a hot fix, and should only contain critical bugs that got released recently. 

Also, the hotfix branch is off of stable and a bit higher in the process document: https://github.com/ionic-team/ionic/pull/17581/files#diff-e31f4b9b148fbe76300904f59f4aa33eR109